components/shared/right-sidebar/SocialMediaLinks.tsx

```diff
@@ -0,0 +1,69 @@
+import Image from "next/image";
+import github from '@/public/socials/github.svg'
+import linkedin from '@/public/socials/linkedin.svg'
+import twitter from '@/public/socials/twitter.svg'
+import instagram from '@/public/socials/instagram.svg'
+import facebook from '@/public/socials/facebook.svg'
+import dribble from '@/public/socials/dribbble.svg'
+
```

Comment:
Hardcoded each of these for now. Had a really hard time mapping over an object and sending the appropriate key, or value to this component as props.

I wasn't able to use the (for..in) loop within my JSX.  I also tried:
```
import github from '@/public/socials/github.svg';
import linkedin from '@/public/socials/linkedin.svg';

const socials = { github: '@nikkyeva', linkedIn: '@nikkyeva' }

Object.keys(socials).map((icon, idx) => {
  return (
      <div icon={idx}>
  <Image
        src={icon}
        width={50}
        height={50}
        alt={`${icon} Icon`}
      />
      <p className='text-white-300 paragraph-2-regular inline-block'>
        {icon[socials]}
      </p>
      </div>
  )
})
```

But it was converting the key to a string (as expected!), so the icon variable github to 'github', so wasn't able to read the source path correctly.  [github]: '@nikkyeva' didn't help either